autotyping/autotyping.py

```diff
@@ -657,3 +688,127 @@ def get_decorator_kind(dec: libcst.Decorator) -> Optional[DecoratorKind]:
         ):
             return DecoratorKind.abstractmethod
     return None
+
+
+# strategy taken from https://github.com/Zac-HD/hypothesis/blob/07ff885edaa0c11f480a8639a75101c6fe14844f/hypothesis-python/src/hypothesis/extra/ghostwriter.py#L319
+# but with more stuff moved up into the _NAMES tuples, since we don't care about ranges
+# and similar
+# fmt: off
```

Comment:
Please remove the fmt off, I'm a Black maintainer and I want all my code to use Black :)

Reply:
I'm a `shed` maintainer, but still prefer to disable one-item-per-line formatting in these cases.  Your library though 🤷‍♂️

Reply:
removing `fmt: off` adds 86 lines to the file currently. I could also clean up the lines slightly, they're currently somewhat hodge-podge since I wanted to initially minimize difference with the original code from hypothesis to make it easier to compare them so I added new ones on their own lines at the end - but easy to sort and squash the lines now.

Reply:
Oh and with 114 lines of constants I'd personally either want to put them at the bottom of the file or excise them to a separate file.

Reply:
A separate file for this sounds good. We're getting to 800 lines now, which is enough that splitting it up makes sense.